### PR TITLE
Add admin.barriers.* APIs #625

### LINKED ISSUE #625: admin.barriers.* APIs
A new set of admin APIs are publicly available now.

* https://api.slack.com/methods/admin.barriers.create
* https://api.slack.com/methods/admin.barriers.delete
* https://api.slack.com/methods/admin.barriers.list
* https://api.slack.com/methods/admin.barriers.update

### Category (place an `x` in each of the `[ ]`)

* [ ] **bolt** (Bolt for Java)
* [ ] **bolt-{sub modules}** (Bolt for Java - optional modules)
* [x] **slack-api-client** (Slack API Clients)
* [ ] **slack-api-model** (Slack API Data Models)
* [ ] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [ ] **slack-app-backend** (The primitive layer of Bolt for Java)

### Requirements

Please make sure if this topic is specific to this SDK. For general questions/issues about Slack API platform or its server-side, could you submit questions at https://my.slack.com/help/requests/new instead. :bow:

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to the those rules.

### PR DESCRIPTION
**Pull request by seratch**
This pull request fixes #625 by adding new endpoints under `admin.barriers` namespace.

### Category (place an `x` in each of the `[ ]`)

* [ ] **bolt** (Bolt for Java)
* [ ] **bolt-{sub modules}** (Bolt for Java - optional modules)
* [x] **slack-api-client** (Slack API Clients)
* [x] **slack-api-model** (Slack API Data Models)
* [ ] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [ ] **slack-app-backend** (The primitive layer of Bolt for Java)

## Requirements

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to the those rules.

### CODE REVIEW

**Comment by seratch** [comment]
Can anyone review this PR? I will be merging this by the end of this week.

File: slack-api-client/src/main/java/com/slack/api/methods/request/apps/event/authorizations/AppsEventAuthorizationsListRequest.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
     /**
      * You'll receive an event_context identifying an event in each event payload sent to your app.
-     *
+     * <p>
```
Comment (stevengill) [inline slack-api-client/src/main/java/com/slack/api/methods/request/apps/event/authorizations/AppsEventAuthorizationsListRequest.java:18]: why the `<p>`

**Review by stevengill — APPROVED** [review]

**Review by misscoded — APPROVED** [review]
One thank you, one nit and one question! From a pattern-matching perspective, all looks to be in order.

File: slack-api-client/src/main/java/com/slack/api/methods/request/apps/event/authorizations/AppsEventAuthorizationsListRequest.java (hunk 1 above)
Comment (seratch) [reply slack-api-client/src/main/java/com/slack/api/methods/request/apps/event/authorizations/AppsEventAuthorizationsListRequest.java:18]: I ran the code formatter to all the code in the same package and the code formatter does it automatically. A blank line is always treated this way.

File: json-logs/samples/api/admin.barriers.create.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ]
+  },
+  "barrier": {
+    "id": "",
+    "enterprise_id": "E00000000",
+    "primary_usergroup": {
+      "id": "S00000000",
+      "name": ""
+    },
+    "barriered_from_usergroups": [
```
[the remark this answers is not in the document]
Comment (seratch) [reply json-logs/samples/api/admin.barriers.create.json:18]: Actually, these files are automatically generated by unit tests. This means these files are evidences I surely tested the code with the APIs.

File: slack-api-client/src/main/java/com/slack/api/methods/impl/AsyncMethodsClientImpl.java [hunk 3]
```diff
@@ -281,6 +289,47 @@ public MethodsClient underlying() {
...
+    }
+
+    @Override
+    public CompletableFuture<AdminBarriersDeleteResponse> adminBarriersDelete(RequestConfigurator<AdminBarriersDeleteRequest.AdminBarriersDeleteRequestBuilder> req) {
+        return adminBarriersDelete(req.configure(AdminBarriersDeleteRequest.builder()).build());
+    }
+
+    @Override
+    public CompletableFuture<AdminBarriersListResponse> adminBarriersList(AdminBarriersListRequest req) {
+
```
[the remark this answers is not in the document]
Comment (seratch) [reply slack-api-client/src/main/java/com/slack/api/methods/impl/AsyncMethodsClientImpl.java:314]: Thanks!

```suggestion
```

File: slack-api-client/src/test/java/test_locally/api/MethodsTest.java [hunk 4]
```diff
@@ -35,15 +35,19 @@ public void verifyTheCoverage() {
         }
 
         String[] allMethodNames = methods.split(",");
-        List<String> excludedMethodNames = Arrays.asList();
+        // TODO: https://github.com/slackapi/java-slack-sdk/issues/634
+        List<String> excludedMethodNames = Arrays.asList(
```
[the remark this answers is not in the document]
Comment (seratch) [reply slack-api-client/src/test/java/test_locally/api/MethodsTest.java:39]: Updating `String methods` with the latest info includes these methods but we are not ready to support these this time. That's why I excluded three endpoints.

**Pull request merged**